apps/taiga-lumbermill/src/dashboards/crypto/components/minter/minter-deploy/minter-deploy.component.html

```diff
@@ -0,0 +1,84 @@
+@let token = minterForm.controls[0].value;
+@let symbol = minterForm.controls[1].value;
+@let amount = minterForm.controls[2].value;
+@let description = minterForm.controls[3].value;
+<div class="info">
+    <div tuiCell="l">
+        <tui-avatar
+            size="xl"
+            [src]="!openIcon && urlIcon !== '' ? urlIcon : '@tui.pencil'"
+            (click)="showDialog()"
+        />
+        <div tuiTitle>
+            {{ token == '' ? 'Jetton Name' : token }} ({{ symbol == '' ? 'Symbol' : symbol }})
+            <div tuiSubtitle>{{ description == '' ? 'Description' : description }}</div>
+        </div>
+    </div>
+</div>
+@for (control of minterForm['controls']; track $index) {
+    <div class="block-input">
+        @if (minterService[$index].type === 'text') {
+            <tui-input [formControl]="control">
+                {{ minterService[$index].title }}
+                <input tuiTextfieldLegacy />
+            </tui-input>
+        } @else {
+            <tui-input-number [formControl]="control">
+                {{ minterService[$index].title }}
+                <input tuiTextfieldLegacy />
+            </tui-input-number>
+        }
+        <div
+            tuiSubtitle
+            [style.padding-left]="'0.4rem'"
+        >
+            {{ minterService[$index].description }}
```

Comment:
let's use hints for this descriptions https://taiga-ui.dev/components/input/API?tuiHintContent=Some%20content